lib/ansible/modules/network/aci/aci_tenant.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_tenant
+short_description: Manage tenants on Cisco ACI fabrics
+description:
+- Manage tenants on a Cisco ACI fabric.
+author:
+- Swetha Chunduri (@schunduri)
+- Dag Wieers (@dagwieers)
+version_added: '2.4'
+requirements:
+- ACI Fabric 1.0(3f)+
+options:
+  method:
+    description:
+    - The HTTP method of the request.
+    - Using C(delete) is typically used for deleting objects.
+    - Using C(get) is typically used for querying objects.
+    - Using C(post) is typically used for modifying objects.
+    required: true
+    default: get
+    choices: [ delete, get, post ]
+    aliases: [ action ]
+  tenant_name:
+    description:
+    - The name of the tenant.
+    required: yes
+  descr:
+    description:
+    - Description for the AEP.
+extends_documentation_fragment: aci
+'''
+
+EXAMPLES = '''
+- name: Add a new tenant
+  aci_tenant:
+    hostname: acme-apic-1
+    username: admin
+    password: SomeSecretPassword
+    tenant_name: production
+    descr: Name of the production tenant
+    state: present
+'''
+
+RETURN = r'''
+status:
+  description: The status code of the http request
+  returned: upon making a successful GET, POST or DELETE request to the APIC
+  type: int
+  sample: 200
+response:
+  description: Response text returned by APIC
+  returned: when a HTTP request has been made to APIC
+  type: string
+  sample: '{"totalCount":"0","imdata":[]}'
+'''
+
+import json
+
+# from ansible.module_utils.aci import aci_argument_spec, aci_login, aci_response
+from ansible.module_utils.basic import AnsibleModule, get_exception
+from ansible.module_utils.urls import fetch_url
+
+aci_argument_spec = dict(
+    hostname=dict(type='str', required=True, aliases=['host']),
+    username=dict(type='str', default='admin', aliases=['user']),
+    password=dict(type='str', required=True, no_log=True),
+    protocol=dict(type='str'),  # Deprecated in v2.8
```

Comment:
Please change to:
`protocol=dict(type='str', removed_in_version='2.6'),  # Deprecated in v2.6`

So this matches https://github.com/ansible/ansible/pull/27070